Lib/fontTools/feaLib/parser.py

```diff
@@ -1169,10 +1169,15 @@ def parse_feature_block_(self):
         location = self.cur_token_location_
         tag = self.expect_tag_()
         vertical = (tag in {"vkrn", "vpal", "vhal", "valt"})
+
         stylisticset = None
         if tag in ["ss%02d" % i for i in range(1, 20+1)]:
             stylisticset = tag
 
+        cv_feature = None
+        if tag in ["cv%02d" % i for i in range(1, 99+1)]:
```

Comment:
this `["cv%02d" % i for i in range(1, 99+1)]` (as well as the similar list comprehension two lines above this) should really be class attributes as they are constant.

Reply:
also, there should some `elif` here, because `tag` can't be all at the same time. If it's `vertical`, then it's not `stylisticset`, if it's `stylisticset` then it's not `cv_features`, etc.